index.js

```diff
@@ -1750,7 +1764,6 @@ export async function createJavaBom(path, options) {
           gradleDepArgs.push(`:${sp.name}:${depTaskWithArgs[0]}`);
         }
       }
-      gradleDepArgs.push("--parallel"); //flag to enable multi-threading
```

Comment:
Adding the `--parallel` flag causes Gradle to output to stdout inconsistently in extremely large (100+ sub-project) projects. Removing the --parallel flag seems to eliminate it, while still giving the performance benefits (since we run a single statement). Unsure why, observed through experiments.